package.json

```diff
@@ -35,6 +35,7 @@
 		"bluebird": "^3.7.2",
 		"chalk": "^1.1.3",
 		"ecmarkdown": "5.0.1",
+		"eslint": "^6.8.0",
```

Comment:
eslint 7 is out, any reason to stick to v6?

Reply:
Because we were already using eslint 6.x as a dev dependency, and I have no interest in dealing with the consequences of bumping that.